pkg/util/byteutil/byteutil_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package byteutil
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMust(t *testing.T) {
```

Comment:
This unit test is fine, because it is a unit test for `Must` function. If we only keep this function, the title of this PR should be changed to "add unit test for `byteutil.Must`"